ImageInNavigationBarDemo/DemoTableViewController.swift

```diff
@@ -7,3 +7,157 @@
 //
 
 import UIKit
+
+class DemoTableViewController: UITableViewController {
+
+    let imageView = UIImageView(image: UIImage(named: "image_name"))
+
+    /// WARNING: Change these constants according to your project's design
+    private struct Const {
+        /// Image height/width for Large NavBar state
+        static let ImageSizeForLargeState: CGFloat = 40
+        /// Spacing from right anchor of safe area to right anchor of Image
+        static let ImageRightSpace: CGFloat = 16
+        /// Spacing from bottom anchor of NavBar to bottom anchor of Image for Large NavBar state
+        static let ImageBottomSpaceForLargeState: CGFloat = 12
+        /// Spacing from bottom anchor of NavBar to bottom anchor of Image for Small NavBar state
+        static let ImageBottomSpaceForSmallState: CGFloat = 6
+        /// Factor used to multiply your ImageSizeForLargeState to get the Image Size for Small NavBar State. For example: Image size for Large NavBar is 40, Image size for Small NavBar is 32, then value of SmallImageFactor = 32/40 = 0.8
+        static let SmallImageFactor: CGFloat = 0.8
+
+        /// Height of NavBar for Small state. Usually it's just 44
+        static let NavBarHeightSmallState: CGFloat = 44
+        /// Height of NavBar for Large state. Usually it's just 96.5 but if you have a custom font for the title, please make sure to edit this value since it changes the height for Large state of NavBar
+        static let NavBarHeightLargeState: CGFloat = 96.5
+    }
+    
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationController?.navigationBar.prefersLargeTitles = true
+
+        title = "Large Title"
+
+        guard let navigationBar = self.navigationController?.navigationBar else { return }
+
+        // Initial setup for image for Large NavBar state since the the screen always has Large NavBar once it gets opened
+        navigationBar.addSubview(imageView)
+        imageView.layer.cornerRadius = 20.0
```

Comment:
`cornerRadius` should be `ImageSizeForLargeState / 2` so that image view will be circled for any constant

Reply:
right, thanks!